tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+skipsdist = True
+envlist = py38, lint, fmt, checkall
+
+[testenv]
+deps = pipenv
+setenv = PYTHONPATH = {toxinidir}
+         PIPENV_VERBOSITY = -1
+commands =
+    pipenv sync --dev
+    bash -c 'tail -n +2 tests/size_list.tsv | xargs -L 1 pipenv run python tests/size_check.py verify'
+
+[testenv:lint]
+deps = pipenv
+setenv = PYTHONPATH = {toxinidir}
+         PIPENV_VERBOSITY = -1
+commands =
+    pipenv sync --dev
+    pipenv run pyright
+    pipenv run flake8 src
+
+[testenv:fmt]
+deps = pipenv
+setenv = PYTHONPATH = {toxinidir}
+         PIPENV_VERBOSITY = -1
+commands =
+    pipenv sync --dev
+    pipenv run isort src
+    pipenv run black src
+
+[testenv:checkall]
+deps = pipenv
+setenv = PYTHONPATH = {toxinidir}
+         PIPENV_VERBOSITY = -1
+commands =
+    pipenv sync --dev
+    pipenv run pyright
+    pipenv run flake8 src
+    pipenv run isort --check-only src
+    pipenv run black --check src
+
+[flake8]
+ignore = E741,W503
+extend-ignore = E203
+exclude = .git,.github,__pycache__,rust,shell,data
+max-line-length = 160
+
+# too high. need to reduce complexity
+max-complexity = 33
```

Comment:
This values is the maximum allowed McCabe complexity value for a block of code.
ref:
https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-max-complexity

Currently I set `33` but it should be 10 or lower.
We need to improve the code.